app/src/main/java/app/insti/fragment/FeedFragment.java

```diff
@@ -92,23 +99,20 @@ public void onPause()
     public void onResume()
     {
         super.onResume();
-        if(index != -1)
-        {
+        if(index != -1) {
             mLayoutManager.scrollToPositionWithOffset( index, top);
         }
     }
 
     private void updateFeed() {
 
         RetrofitInterface retrofitInterface = ((MainActivity) getActivity()).getRetrofitInterface();
-        retrofitInterface.getNewsFeed("sessionid=" + getArguments().getString(Constants.SESSION_ID)).enqueue(new Callback<NewsFeedResponse>() {
+        retrofitInterface.getNewsFeed(((MainActivity)getActivity()).getSessionIDHeader()).enqueue(new Callback<NewsFeedResponse>() {
```

Comment:
Consider moving MainActivity$getSessionIDHeader() to Helpers

Reply:
Done